SuperBuild/External_CTKAPPLAUNCHER.cmake

```diff
@@ -22,15 +22,20 @@ if(Slicer_USE_CTKAPPLAUNCHER)
   endif()
 
   if(NOT DEFINED CTKAppLauncher_DIR)
-
     SlicerMacroGetOperatingSystemArchitectureBitness(VAR_PREFIX CTKAPPLAUNCHER)
     set(launcher_version "0.1.27")
+    set(repo commontk)
     # On windows, use i386 launcher unconditionally
     if("${CTKAPPLAUNCHER_OS}" STREQUAL "win")
       set(CTKAPPLAUNCHER_ARCHITECTURE "i386")
       set(md5 "3f05dcc605ac2144edc69b28c27bb8d1")
     elseif("${CTKAPPLAUNCHER_OS}" STREQUAL "linux")
-      set(md5 "a9a8aab9c0e91cdd0b5265eb799daf74")
+      if("${CTKAPPLAUNCHER_ARCHITECTURE}" STREQUAL "aarch64")
+        set(md5 "54afe832001fb0c0146c089eeccd2d8f")
+        set(repo Slicer)
```

Comment:
How much work would it be to support building the official launcher to support aarch64 ? Did you build Qt statically for that platform ? I wonder if we could leverage the `dockcross/linux-arm64` image. See https://github.com/dockcross/dockcross#cross-compilers

Reply:
Depends on the build environment of ctkapplauncher. It wasn't much work for me. I did built Qt statically.

Reply:
The launcher is built by Slicer superbuild, but if the user uses dynamic Qt, it can't be used to generate the Slicer Python executable.